lib/iris/tests/unit/cube/test_CubeList.py

```diff
@@ -301,5 +302,18 @@ def test_summary_stash(self):
         self.assertEqual(str(self.cubes), expected)
 
 
+class TestRealiseData(tests.IrisTest):
+    def test_realise_data(self):
+        # Simply check that calling CubeList.realise_data is calling
+        # _lazy_data.co_realise_cubes.
+        mock_cubes_list = [mock.Mock(ident=count) for count in range(3)]
+        test_cubelist = CubeList(mock_cubes_list)
+        call_patch = self.patch('iris.cube._lazy.co_realise_cubes')
```

Comment:
You might find that you can now mock the actual function ```iris._lazy_data.co_realise_cubes``` as a result of the changed import semantics.

Reply:
Good spot :+1: